jsapp/js/account/plan.component.tsx

```diff
@@ -0,0 +1,442 @@
+import React, {useEffect, useReducer, useState} from 'react';
+import styles from './plan.module.scss';
+import {
```

Comment:
When importing types please use `import type` and split this into two imports

Reply:
Why split it into two imports?